### PR TITLE
Implement connect attributes

### PR DESCRIPTION
**Pull request by meeuw**
Implement connect attributes

The connection attributes can be queried using:
```select * from performance_schema.session_connect_attrs```

### CODE REVIEW

**Comment by meeuw** [comment]
@methane thanks for your review! I've fixed your suggestions.

File: pymysql/connections.py [hunk 1]
```diff
@@ -676,6 +676,9 @@ def _config(key, arg):
         client_flag |= CLIENT.CAPABILITIES
         if self.db:
             client_flag |= CLIENT.CONNECT_WITH_DB
+
+        client_flag |= CLIENT.CONNECT_ATTRS
+
```
Comment (methane) [inline pymysql/connections.py]: Add CONNECT_ ATTRS to CLIENT.CAPABILITIES.

File: pymysql/connections.py [hunk 2]
```diff
@@ -698,6 +701,18 @@ def _config(key, arg):
...
+        self._connect_attrs = {
+            '_client_name': 'pymysql',
+            '_pid': str(os.getpid()),
+            '_client_version': VERSION_STRING,
+        }
+
+        if program_name:
+            self._connect_attrs["program_name"] = program_name
+        else:
+            self._connect_attrs["program_name"] = sys.argv[0]
```
Comment (methane) [inline pymysql/connections.py]: elif sys.argv:

**Comment by meeuw** [comment]
also fixed

**Comment by meeuw** [comment]
Could you please merge this PR and make a new release?

**Comment by methane** [comment]
I need to do more work for next release.

**Comment by meeuw** [comment]
@methane thanks a lot! Please let me know if I can help you with your work for the next release.

**Pull request merged**